### PR TITLE
script for package versions

### PR DESCRIPTION
**Pull request by modship**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

Proposal : 

if we consider that the versions >= 1 are MainNet and versions <1 are TestNet / SandBox.
Parsing the current `VERSION` in `massa-models/src/config/constants` and for now, remove `TEST` or `SAND` from it to replace by `0`. 
That give for example `0.23.0`.
Then replace in all `Cargo.toml` the package version by this version.

<details>
<summary>Deprecated</summary>

Actually the script is in build.rs file which is a pre-build script. It's fast but maybe it's "overkill" to run it at every build ? 

</details>

A new massa-package called `massa-xtask` was created with this `main.rs`
```rust
/// to use it task: cargo xtask <task_name>
/// example: cargo xtask update_package_versions to update package versions
fn main() {
    let task = env::args().nth(1);

    match task.as_deref() {
        // We can add more tasks here
        Some("update_package_versions") => update_package_versions(),
        _ => panic!("Unknown task"),
    }
}
```

We can run the task with `cargo xtask update_package_versions`. With that it's possible to add easily some other task.

Example : 

- update `VERSION` to `TEST.24.0` in `massa-models/src/config/constants.rs` 
- call `cargo xtask update_package_versions`

image

### CODE REVIEW

**Comment by modship** [comment]
> We shouldn't run it on every build I think because it will trigger on the node runners build. Maybe we can use cargo script to run it whenever we want

yes i think too

File: massa-node/Cargo.toml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 [package]
-name = "massa-node"
-version = "0.1.0"
+name = "massa_node"
```
Comment (AurelienFT) [inline massa-node/Cargo.toml:2]: This change and the one the client change the name of the binary also ?

Comment (modship) [reply massa-node/Cargo.toml:2]: yes, the binary was renamed. I rollback my modification on it for now

Comment (AurelienFT) [reply massa-node/Cargo.toml]: Ok yes thank you because it implies a lot for a lot of tooling etc

**Review by AurelienFT — APPROVED** [review]
Perfect thanks a lot !

File: massa-node/Cargo.toml (hunk 1 above)
Comment (modship) [reply massa-node/Cargo.toml]: Yes of course I had to check

**Pull request merged**